### ISSUE TITLE
Download Progress is always displayed as 0 bytes

### ISSUE THREAD

**Issue by OscarVanL** [plexinc/plex-media-player#933]
When downloading an item from my Plex server, the Downloads window shows the download progress as 0 bytes. This does not go up throughout the download.
![Capture](https://user-images.githubusercontent.com/7999692/71772375-0b2bfa80-2f42-11ea-8ddb-2f6ee62dd195.PNG)

When the download is finished it updates to display the correct amount.

I have not included a video sample as this happens on every film and TV show I've tried downloading. I can provide one if it is desired.

### Test environment

**PMS Version:** Version 1.18.3.2156 
**PMP Version:** Version 1.4.1.940-574c2fa7
**Platform (Windows/macOS/Embedded RPi/Embedded Intel):** Windows

### Steps to reproduce
1. Download something from your Plex library
2. Enter the Downloads window

### Current behavior
Downloaded remains at 0 bytes throughout download.

### Expected behavior
Downloaded should increase as the films download.

**Comment by mseeley** [comment]
You're right, the download total is only updated as downloads complete or are removed. This is working as intended but, you're right, I've never noticed how odd it looks. Now I can't unsee it. :)

Asking colleagues for input.

**Comment by OscarVanL** [comment]
I suspected this might have been the case. From a UX perspective It's confusing if you've got sufficiently large films that the progress circle fills up slowly, because you're unsure if the download has started or not.
